30-remote-site-setup.sh

```diff
@@ -153,7 +158,18 @@ fi
 echo "Using Bosco tarball: $(bosco_findplatform --url)"
 for ruser in $users; do
     setup_ssh_config
-    [[ $cvmfs_wn_client == 'no' ]] && setup_endpoints_ini
+done
+
+###################
+# REMOTE COMMANDS #
+###################
+
+# We have to pick a user for SSH, may as well be the first one
+remote_os_info=$(fetch_remote_os_info "${users%%[:blank:]*}" "$remote_fqdn")
```

Comment:
The `${%%}` syntax is a glob, not a regex, so this doesn't work. Example:
```
$ users='  osg01   osg02    '
$ echo "X${users%%[:blank:]*}X"
X  osg01   osg02    X
```

Reply:
Dang, I could've sworn I had it working in my testing